### PR TITLE
⚙️ Enhancements For Loyalty

### PR DESCRIPTION
**Pull request by jalmatari**

### CODE REVIEW

File: src/views/components/header/breadcrumbs.twig [hunk 1]
```diff
@@ -6,6 +6,7 @@
 | breadcrumbs[].url   | string             | A url of breadcrumb                   |
 #}
 
+{{dd(breadcrumbs)}}
```
Comment (jalmatari) [inline src/views/components/header/breadcrumbs.twig:9]: 🌚

File: src/views/pages/loyalty.twig [hunk 2]
```diff
@@ -150,34 +159,34 @@
...
-                                        <img class="lazy-load sm:h-full w-full object-cover"
+                                        <img class="lazy-load sm:h-full w-full object-cover bg-gray-100 rounded-t-lg"
                                              src="{{ 'images/s-empty.png' | asset }}"
                                              data-src="{{ prize.image }}"
                                              alt="{{ prize.name }}"/>
                                     {% endset %}
-                                    {% if prize.url %}
-                                        <a href="{{ prize.url }}" class='product-entry__image h-40'>
+                                    {% if prize.prize_url %}
+                                        <a href="{{ prize.prize_url }}">
```
Comment (jalmatari) [inline src/views/pages/loyalty.twig:175]: 🤨

File: src/assets/js/loyalty.js [hunk 3]
```diff
@@ -4,38 +4,34 @@ import Slider from './partials/slider'
...
-                return i * 100;
-            },
-        }, '-=3200')
+        })
+        // .add({
+        //     targets: '.btn-anime',
+        //     opacity: [0, 1],
+        //     duration: 2000,
+        //     translateX: [20, 0]
+        // })
```
Comment (jalmatari) [inline src/assets/js/loyalty.js:34]: if there is no need remove it

File: src/views/pages/loyalty.twig [hunk 4]
```diff
@@ -150,34 +159,34 @@
...
                                         </a>
                                     {% else %}
                                         {{ img }}
                                     {% endif %}
                                     <div class="flex-1 p-5">
                                         <h3 class="product-entry__title leading-6 mb-2.5">
-                                            {% if prize.url %}
-                                                <a href="{{ prize.url }}">{{ prize.name }}</a>
+                                            {% if prize.prize_url %}
+                                                <a href="{{ prize.prize_url }}">{{ prize.name }}</a>
```
Comment (jalmatari) [inline src/views/pages/loyalty.twig:184]: 🙂

File: src/views/pages/loyalty.twig [hunk 5]
```diff
@@ -130,15 +139,15 @@
         </section>
 
         {# 2-block:: prizes #}
-        {% for prize_group in loyalty.prizes %}
-            <section class="mb-28 last:mb-0" id="loyalty-slider-{{ prize_group.type }}">
+        {% for prize in loyalty.prizes %}
+            <section id="loyalty-slider-{{ prize.type }}">
```
Comment (jalmatari) [inline src/views/pages/loyalty.twig:143]: use `prize_group`

File: src/views/components/header/breadcrumbs.twig (hunk 1 above)
Comment (ashrafreda) [reply src/views/components/header/breadcrumbs.twig:9]: done

File: src/views/pages/loyalty.twig (hunk 2 above)
Comment (ashrafreda) [reply src/views/pages/loyalty.twig:175]: there is a prize_url at all?

File: src/assets/js/loyalty.js (hunk 3 above)
Comment (ashrafreda) [reply src/assets/js/loyalty.js:34]: ok

File: src/views/pages/loyalty.twig (hunk 2 above)
Comment (ashrafreda) [reply src/views/pages/loyalty.twig]: done

File: src/views/pages/loyalty.twig (hunk 4 above)
Comment (ashrafreda) [reply src/views/pages/loyalty.twig]: done

File: src/views/pages/loyalty.twig (hunk 5 above)
Comment (ashrafreda) [reply src/views/pages/loyalty.twig]: done

**Comment by SallaDev** [comment]
https://github.com/SallaApp/theme-raed/pull/90 ` is been merged ✅`

-------------------

**Pull request merged**